test/Microsoft.AspNetCore.Mvc.Core.Test/Infrastructure/DefaultActionSelectorTests.cs

```diff
@@ -174,6 +174,82 @@ public void SelectCandidates_NoMatch_ExcludesAttributeRoutedActions()
         }
 
         [Fact]
+        public void SelectCandidates_Match_CaseSensitiveMatch_IncludesAllCaseInsensitiveMatches()
```

Comment:
I'll add a non-related AD to both of these tests so we can see that it's not included.